### ISSUE TITLE
[FEATURE REQUEST] Style/SingleSpaceBeforeFirstArg:AllowForAlignment

### ISSUE THREAD

**Issue by aeberlin** [bbatsov/rubocop#2504]
There exists this configuration option for `Style/ExtraSpacing`:

```yaml
Style/ExtraSpacing:
  AllowForAlignment: true
```

Can we add the same configurability for `Style/SingleSpaceBeforeFirstArg` ?

Useful example:

```haml
.user-form
  = simple_form_for User.new do |f|
    = f.inline_input   :full_name,     as: :string
    = f.disabled_input :password,      as: :password
    = f.masked_input   :zip_code,      as: :string
    = f.masked_input   :email_address, as: :email
    = f.masked_input   :phone_number,  as: :tel
```

**Comment by alexdowad** [comment]
Just pushed a fix for this one.

**Comment by aeberlin** [comment]
Awesome! Thanks!

**Comment by aeberlin** [comment]
@alexdowad I've been getting this warning, but I'm using the most recent release (0.36)

`Warning: unrecognized cop Style/SingleSpaceBeforeFirstArg found in foo/.rubocop.yml`

Thoughts? Thanks

**Comment by alexdowad** [comment]
It's called `SpaceBeforeFirstArg` now. If my open PR is merged, RC will print a more helpful message. (See 6397b14.)

**Comment by aeberlin** [comment]
Ahhh! Gotcha, thanks!
